cypress/tests/tier1/vm-actions/right-click.cy.ts

```diff
@@ -1,21 +1,25 @@
-import { VM_STATUS } from '../../../utils/const/index';
-import { VM_IT_QUICK, VM_TMPL_QUICK } from '../../../utils/const/testVM';
+import { TEST_NS, VM_STATUS } from '../../../utils/const/index';
+import { VM_IT_CUST, VM_TMPL_CUST } from '../../../utils/const/testVM';
 import { waitForStatus } from '../../../views/vm-flow';
 
+const VM_NAMES = [VM_IT_CUST.name, VM_TMPL_CUST.name];
+
 describe('Right-click of project/folder', () => {
   before(() => {
-    cy.visitVMs();
+    cy.startVM(VM_NAMES);
+    cy.beforeSpec();
+    cy.visitVMsVirt();
   });
 
-  it('start two VMs for test', () => {
-    cy.startVM(VM_IT_QUICK.name);
-    cy.startVM(VM_TMPL_QUICK.name);
+  after(() => {
+    cy.stopVM(VM_NAMES);
   });
 
   it('stop vms by right-click of project', () => {
-    cy.contains('button.pf-v6-c-tree-view__node-text', 'default', { timeout: 180000 }).rightclick();
+    cy.get(`[data-test-id=${VM_IT_CUST.name}]`).should('be.visible');
```

Comment:
`byLegacyTestID` ? i can see that we can use `byLegacyTestID` multiple times in this pr